hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmCreateOperation.java

```diff
@@ -44,9 +44,16 @@ protected boolean isEnabled() {
 
     @Override
     protected Address targetContractAddress(MessageFrame frame) {
-        final var updater = (HederaEvmWorldUpdater) frame.getWorldUpdater();
-        final Address address = updater.newContractAddress(frame.getRecipientAddress());
+        final var sourceAddressOrAlias = frame.getRecipientAddress();
+        final var sourceNonce =
+                frame.getWorldUpdater().getAccount(sourceAddressOrAlias).getNonce();
+        final var updater = (HederaEvmStackedWorldUpdater) frame.getWorldUpdater();
+        // Decrement nonce by 1 to normalize the effect of transaction execution
```

Comment:
This comment about the decrement - is this because the call here takes place _after_ the transaction has done something that incremented the nonce?  Maybe a different wording than "normalize the effect" could make it clearer (at least to me)?

(Oh, NVM, I see the same comment at `org.hyperledger.besu.evm.operation.CreateOperation.targetContractAddress` - still makes me feel uneasy, as if there should have been a better way, in BESU, to deal with this.)